ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,125 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func curlRayServicePod(
+	t Test,
+	rayService *rayv1.RayService,
+	curlPod *corev1.Pod,
+	curlPodContainerName, //nolint:unparam // This parameter is necessary TODO: remove it when calls this function with a different curlPodContainerName
+	rayServicePath,
+	body string,
+) (bytes.Buffer, bytes.Buffer) {
+	cmd := []string{
+		"curl",
+		"-X", "POST",
+		"-H", "Content-Type: application/json",
+		fmt.Sprintf("%s-serve-svc.%s.svc.cluster.local:8000%s", rayService.Name, rayService.Namespace, rayServicePath),
+		"-d", body,
+	}
+
+	return ExecPodCmd(t, curlPod, curlPodContainerName, cmd)
+}
+
+func rayServiceSampleYamlApplyConfiguration() *rayv1ac.RayServiceSpecApplyConfiguration {
+	return rayv1ac.RayServiceSpec().WithServeConfigV2(`applications:
+      - name: fruit_app
```

Comment:
can we remove the unnecessary parts of `ServeConfigV2`? For example, `OrangeStand`.

Reply:
I’ve tried several times, and it turns out that at least two fruit stands need to be retained; otherwise, it will keep getting stuck at "wait for deployment ready."

Reply:
> at least two fruit stands need to be retained; otherwise, it will keep getting stuck at "wait for deployment ready."

Interesting. Could you open an issue about the reproduction? We don't need to handle the issue in this PR, but we should track the progress.

> There are three types of fruits here. Do we want to remove one, or keep them all?

Remove one. Just try to make this AC as short as possible.

Reply:
OK, I create one #2557